attributes/default.rb

```diff
@@ -1,6 +1,7 @@
 node.default['qa-chef-server-cluster']['chef-provisioner-key-name'] = 'insecure-chef-provisioner'
 
 node.default['qa-chef-server-cluster']['aws']['region'] = 'us-west-2'
+node.default['qa-chef-server-cluster']['aws']['availability_zone'] = 'b'
```

Comment:
This is a little knit-picky, so feel free to ignore. But my understanding is that ((ped|sem)antically) a region is something like "us-west-2" and an availability zone includes the region "us-west-2b" ?